include/swift/Frontend/FrontendOptions.h

```diff
@@ -195,6 +195,12 @@ class FrontendOptions {
     EmitObject, ///< Emit object file
   };
 
+  bool isCreatingSIL() { return RequestedAction >= EmitSILGen; }
+  bool shouldImportSwiftOnoneModuleIfNoneOrImplicitOptimization() {
```

Comment:
This encodes too much policy in the function name. At best, it should be a static function at the place where it's used; it's an arbitrary set of conditions and I'm not sure why the original author picked these. (Possibly Roman?) Actually changing the behavior can come later, but assigning meaning to it in FrontendOptions is moving in the wrong direction, IMO.

Reply:
I understand the issue, & have made the change into a static function. But the code does get a bit cumbersome that way.